.eslintrc.babel.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
```

Comment:
Babel-specific configuration was pulled out into its own file from `.eslintrc`, but there should be no changes with what was there before.